src/components/label-pill/LabelPill.scss

```diff
@@ -1,13 +1,19 @@
 @import '../../styles/variables';
 
 .bdl-LabelPill {
-    display: inline-block;
+    padding: 3px;
     color: $bdl-gray;
+    font-size: 10px;
+    line-height: 10px;
     background-color: $bdl-gray-10;
     border: 1px solid;
     border-color: $bdl-gray-20;
     border-radius: 10px;
 
+    & + .bdl-LabelPill {
```

Comment:
I believe we rolled back something like this for Buttons because the increased specificity of the selector made it hard to manual adjust spacing

Reply:
i'd be a little less worried about this one, only because it's for multiple instances of this component next to one another. except for this edge case, it doesn't do any external spacing .

Unless that button use case was also its positioning relative to other buttons...

Reply:
It was for 2+ buttons next to each other; it created a situation similar to the other issue with button's :hover/:focus states, where every time you want to change the default, you need to recreate the selector of make one of higher specificity.

Reply:
May also be an issue if you have several pills in a line that wraps, where the wrapped ones have an extra left margin. I'm not sure the ideal way to handle that though, it may end up occurring anyways if spacing was implemented by consumer.

Reply:
I hardened the "between spacing" and will have this up in the next push. using `:last-child` to cancel margin-right